Popup/Popup.less

```diff
@@ -31,9 +32,10 @@
 			.body {
 				padding: @agate-popup-body-padding;
 				margin: @agate-popup-body-margin;
-				border: @agate-popup-body-border-width solid transparent;
+				border: @agate-popup-border;
```

Comment:
This variable should be `@agate-popup-body-border` so it doesn't conflict with (or have to be identical to) `@agate-popup-border`.